src/Worker/Delivery.php

```diff
@@ -278,7 +278,7 @@ private static function setFailedQueue(string $cmd, array $item)
 	private static function deliverDFRN(string $cmd, array $contact, array $owner, array $items, array $target_item, bool $public_message, bool $top_level, bool $followup, int $server_protocol = null)
 	{
 		// Transmit Diaspora reshares via Diaspora if the Friendica contact support Diaspora
-		if (Diaspora::getReshareDetails($target_item ?? []) && !empty(FContact::getByURL($contact['addr'], false))) {
+		if (Diaspora::getReshareDetails($target_item ?? []) && DI::dsprContact()->existsByUri(new Uri($contact['url']))) {
```

Comment:
Please use the function to fetch the Diaspora-Contact by their handle here.

Reply:
Done.